projects/components/src/header/application/application-header.component.ts

```diff
@@ -30,10 +36,17 @@ export class ApplicationHeaderComponent {
   @Input()
   public showTimeRange: boolean = true;
 
+  public pageLevelTimeRangeDisabled$: Observable<boolean>;
+
   public constructor(
     @Inject(GLOBAL_HEADER_HEIGHT) public readonly height: string,
-    private readonly navigationService: NavigationService
-  ) {}
+    private readonly navigationService: NavigationService,
+    private readonly featureStateResolver: FeatureStateResolver
+  ) {
+    this.pageLevelTimeRangeDisabled$ = this.featureStateResolver
```

Comment:
Inverted FF logic here so that two time ranges are not shown on screen at once. Time range is hidden here when FF is Enabled and shown when disabled.

Reply:
We should move the invertFF logic to ht. @aaron-steinfeld